core/src/test/javascript/reportviewer-prompt-spec.js

```diff
@@ -496,14 +495,15 @@ define(["reportviewer/reportviewer-prompt", "reportviewer/reportviewer-logging",
           expect(param.forceUpdate).toBe(undefined);
         });
 
-        it("should not force an update if the datepicker value is not changed, no timezone", function () {
+        // TODO review and fix/remove - started failing after issue [BISERVER-14919]
+        xit("should not force an update if the datepicker value is not changed, no timezone", function () {
```

Comment:
I think we should likely keep these tests failing until solved. Maybe create a "test regression" story associated with `BISERVER-14919` and try to contact the involved team.